tests/utils/pv.go

```diff
@@ -78,6 +80,46 @@ func NewPVDefinition(pvName string, size string, labels map[string]string, stora
 	}
 }
 
+// NewBlockPVDefinition creates a PV definition with volueMode 'Block'
+func NewBlockPVDefinition(pvName string, size string, labels map[string]string, storageClassName string) *k8sv1.PersistentVolume {
+	volumeMode := k8sv1.PersistentVolumeBlock
+	return &k8sv1.PersistentVolume{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   pvName,
+			Labels: labels,
+		},
+		Spec: k8sv1.PersistentVolumeSpec{
+			AccessModes:                   []k8sv1.PersistentVolumeAccessMode{k8sv1.ReadWriteOnce},
+			PersistentVolumeReclaimPolicy: k8sv1.PersistentVolumeReclaimDelete,
+			Capacity: k8sv1.ResourceList{
+				k8sv1.ResourceName(k8sv1.ResourceStorage): resource.MustParse(size),
+			},
+			StorageClassName: storageClassName,
+			VolumeMode:       &volumeMode,
```

Comment:
VolumeMode: &k8sv1.PersistentVolumeBlock,
doesn't work?

Reply:
nope. it complains: "cannot take the address of "kubevirt.io/containerized-data-importer/vendor/k8s.io/api/core/v1".PersistentVolumeBl......."

Reply:
Interesting, its sometimes hard to figure out what you can and cannot get a pointer to with &